### PR TITLE
[WIP] Testing user model including validations with rspec 

### PR DESCRIPTION
**Pull request by Weltinio**
- Added first_name, last_name, and date-of-birth columns to Users
109810149-1ffa4000-7c64-11eb-892a-9a35f22c9e17

- Made validations for the added columns
- created rspec tests for the added columns' validations
- need to figure out the issue with age validation.
109809966-e7f2fd00-7c63-11eb-804f-4e378d7f19d5

`What's missing?`
- Legal age validation.

### CODE REVIEW

File: spec/factories/user.rb [hunk 1]
```diff
@@ -4,6 +4,6 @@
     password {"123456"}
 		first_name { "Test" }
 		last_name { "Tester" }
-		age { 19 }
+		date_of_birth { "16/05/1993" }
   end
 end
```
Comment (analyn34) [inline spec/factories/user.rb:9]: fix code alignment, use 2 spaces

File: app/models/user.rb [hunk 2]
```diff
@@ -8,19 +8,19 @@ class User < ApplicationRecord
...
-    if date_of_birth.present? && date_of_birth > 18.years.ago.to_d
-      errors.add('You should be 18 years or older.')
-    end
-  end
+  # private
+  # def validate_age
+  #   if date_of_birth.present? && date_of_birth > 18.years.ago.to_s
+  #     errors.add('You should be 18 years or older.')
+  #   end
+  # end
```
Comment (analyn34) [inline app/models/user.rb:24]: remove unnecessary comments

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -32,17 +32,16 @@
...
+    # it "Shoud not save if age is < 18" do
+    #   @user = User.new()
+    #   @user.email = 'testing@email.com'
+    #   @user.password = '123456'
+    #   @user.last_name = 'Tester'
+    #   @user.first_name = 'Test'
+    #   @user.date_of_birth = '03/12/2015'
+
+    #   expect(@user).to_not be_valid
+    # end
```
Comment (analyn34) [inline spec/models/user_spec.rb:44]: remove unnecessary comments

**Comment by analyn34** [comment]
add `What's missing` on your PR description if there's anything that you've missed out

File: app/models/user.rb (hunk 2 above)
Comment (Weltinio) [reply app/models/user.rb]: Done!

**Comment by jeleencubillas** [comment]
Looks good!

**Pull request merged**